target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-wr-1.dts

```diff
@@ -36,7 +36,8 @@
 		pinctrl-names = "default";
 
 		led_power: power {
-			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
```

Comment:
There are at least some unnecessary changes, creating duplication, as the items already exist:

```
		led_power: power {
-			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_POWER;
			gpios = <&tlmm 0 GPIO_ACTIVE_HIGH>;
			color = <LED_COLOR_ID_BLUE>;
			function = LED_FUNCTION_POWER;
```